tools/ci/build-test-ubuntu-with-latest-clang.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+# Copyright (c) 2021 by Apex.AI Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# This script builds iceoryx_hoofs und iceoryx_posh and executes all tests
+
+set -e
+
+DIR="$(cd -P "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+msg() {
+    printf "\033[1;32m%s: %s\033[0m\n" ${FUNCNAME[1]} "$1"
+}
+
+# go to repo root
+ROOT=${DIR}/../..
+cd ${ROOT}
+
+msg "installing build dependencies"
+sudo apt-get update && sudo apt-get install -y libacl1-dev libncurses5-dev
+
+msg "installing latest stable clang"
+wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
```

Comment:
Please see https://github.com/eclipse-iceoryx/iceoryx/pull/897 for the latest changes.

Reply:
done